src/components/calcite-panel/calcite-panel.tsx

```diff
@@ -61,6 +62,11 @@ export class CalcitePanel {
    */
   @Prop({ reflect: true }) dismissible = false;
 
+  /**
+   * Number at which section headings should start for this component.
+   */
+  @Prop() headingLevel: HeadingLevel = 3;
```

Comment:
Do you think we would need to expose the level as a shared resource? The only use cases I can think of right now is for testing and possibly keeping levels in sync between components.